tests/manual_test_create_unsafe_vault.py

```diff
@@ -64,6 +63,18 @@ def r(value, decimals=1):
 logging.info(f"{'':<7} duty={mcd.jug.duty(ilk)} min_amount={token.min_amount}")
 
 
+if osm_price == Wad(0) and isinstance(collateral.pip, OSM):
+    logging.warning("OSM price is 0; poking OSM")
+    osm: OSM = collateral.pip
+    osm.poke().transact()
```

Comment:
I know user might need to wait an hour and poke again to test, but better than nothing.

Reply:
I love that you added this.